packages/plugins/plugin-assistant/src/AssistantPlugin.tsx

```diff
@@ -95,4 +99,12 @@ export const AssistantPlugin = () =>
       activatesBefore: [Events.SetupArtifactDefinition],
       activate: ReactSurface,
     }),
+    defineModule({
+      id: `${meta.id}/module/ai-service`,
+      activatesOn: Events.Startup,
```

Comment:
does this cause Composer to completely re-render when turning the plugin on/off? Generally I've been trying to avoid using this event unless strictly necessary, particularly for user-toggleable plugins

Reply:
What's a better event for this?